src/Suggest/Phrase.php

```diff
@@ -162,18 +162,15 @@ public function addCandidateGenerator(AbstractCandidateGenerator $generator): Ph
     public function toArray(): array
     {
         $array = parent::toArray();
-
         $baseName = $this->_getBaseName();
 
         if (isset($array[$baseName]['candidate_generator'])) {
             $generators = $array[$baseName]['candidate_generator'];
             unset($array[$baseName]['candidate_generator']);
 
             foreach ($generators as $generator) {
-                $keys = \array_keys($generator);
-                $values = \array_values($generator);
-
-                $array[$baseName][$keys[0]][] = $values[0];
+                $key = \array_key_first($generator);
```

Comment:
Let's use array_key_first as the polyfill for PHP 7.3 is present in the dependencies